book/build-service.md

```diff
@@ -0,0 +1,142 @@
+# Build Service
+
+
+## Overview
+
+The Build Service is composed of controllers that create and configure build pipelines. The main input for the Build Service is a Component CR managed by the [Hybrid Application Service (HAS)](hybrid-application-service.md).
+
+![](../diagrams/build-service/build-service.drawio.svg)
```

Comment:
I think the green and red arrows in the diagram may be reversed. Green is default and red is custom?

Reply:
It's written in diagram at bottom

Reply:
ok, I cannot read, fixing ;-)